wandb/sdk/wandb_run.py

```diff
@@ -3206,27 +3254,33 @@ def link_model(
             run.link_model(
                 path="/local/directory",
                 registered_model_name="my_entity/my_project/my_reg_model",
-                model_name="my_model_artifact",
+                name="my_model_artifact",
+                aliases=["production"],
+            )
+
+            run.link_model(
+                path="/local/directory",
+                registered_model_name="my_reg_model",
```

Comment:
can we use a registered model name representative of an ML task, such as Review Summarization, or Object Detection, etc?

Reply:
That would just go against the pattern we have for the other run apis. This is clear so why is it necessary? I don't think the docstrings need to be as descriptive as the main docs.